From a6937272a91ccf946f786377fdffd0cadb812b0b Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Wed, 10 Oct 2018 09:36:28 +0300
Subject: [PATCH] Makefile [pldm-hex]: fix install

Need a recursive make so that correct HL_DEV will be used
---

diff --git a/Makefile b/Makefile
index 29dc6e2..1b294d2 100644
--- a/Makefile
+++ b/Makefile
@@ -220,8 +220,10 @@
 PLDM_HEX_DIR ?= $(IMAGES_DIR)/tools/hl_shared_dir/pldm
 ${PLDM_HEX_DIR}:
 	@mkdir -p $@
-goya-pldm-install gaudi-pldm-install: ${PLDM_HEX_DIR}
+goya-pldm-install-hex gaudi-pldm-install-hex: ${PLDM_HEX_DIR}
 	-cp $(IMAGES_DIR)/$(HL_DEV)-spi-flash.hex $</
+goya-pldm-install gaudi-pldm-install:
+	@$(MAKE) $@-hex
 goya-asic-install:
 	@
 
